fdbserver/RestoreController.actor.cpp

```diff
@@ -115,7 +115,7 @@ ACTOR Future<Void> startRestoreController(Reference<RestoreWorkerData> controlle
 		// recruitRestoreRoles must come after controllerWorker has finished collectWorkerInterface
 		wait(recruitRestoreRoles(controllerWorker, self));
 
-		self->addActor.send(updateHeartbeatTime(self));
+		// self->addActor.send(updateHeartbeatTime(self));
```

Comment:
I chose to comment this out instead of removing it on purpose. This is supposed to track when a role is not responding quickly. I'm thinking about changing it to FDB's canonical way to do that. But it's not the focus of this PR.